CONTRIBUTING.md

```diff
@@ -65,6 +65,10 @@ Other Tips
 
         $ git commit --amend
 
+  - Then force push your branch:
```

Comment:
See my above comment.  I will probably close this and add some updated text directly, but thanks for your effort!